src/main/scala/akka/stream/alpakka/xlsx/SstStreamer.scala

```diff
@@ -1,37 +1,57 @@
 package akka.stream.alpakka.xlsx
 
-import java.io.InputStream
 import java.util.zip.ZipFile
 
 import akka.stream.Materializer
 import akka.stream.alpakka.xml.scaladsl.XmlParsing
-import akka.stream.alpakka.xml.{ Characters, EndElement, ParseEvent, StartElement }
-import akka.stream.scaladsl.{ Keep, Sink, Source, StreamConverters }
+import akka.stream.alpakka.xml.{Characters, EndElement, ParseEvent, StartElement}
+import akka.stream.contrib.ZipInputStreamSource.ZipEntryData
+import akka.stream.scaladsl.{Keep, Sink, Source, StreamConverters}
+import akka.util.ByteString
 
 import scala.concurrent.Future
 
 object SstStreamer {
 
+  private final val EntryName = "xl/sharedStrings.xml"
+  private val defaultSink = Sink.fold[Map[Int, String], (Int, String)](Map.empty)((v1, v2) => v1 + v2)
```

Comment:
If you can afford creating all these intermediate collections, then a simple improvement would be to just:
```scala
Sink.seq[(Int, String)].mapMaterializedValue(_.map(_.toMap)(ExecutionContext.fromExecutor(_.run())))
```

otherwise I'd play around with a builder:
```scala
Sink.fold[mutable.Builder[(Int, String), Map[Int, String]], (Int, String)](Map.newBuilder[Int, String])(_ += _)
  .mapMaterializedValue(_.map(_.result())(ExecutionContext.fromExecutor(_.run())))
```

also this lambda could be easily replaced by `_ + _`